src/components/JsonEditor/JsonEditor.tsx

```diff
@@ -2,16 +2,39 @@ import React, { useEffect } from 'react';
 import classes from './JsonEditor.module.scss'
 import mauiA from '../../config/MauiA.json';
 import { JsonEditorContainer } from "../JsonEditorContainer/JsonEditorContainer";
-import { addNode, getAllNodes, loadSchema, removeNode } from '../../validator/Validator';
+import { getAllNodes, loadSchema, addNode } from '../../validator/Validator';
+import { Template } from 'jsoneditor';
+
+const generateCaption = (key: string) => {
```

Comment:
I feel, display the key as it is also ok

Reply:
This function does not use for now